Platform/OpenCore/OpenCore.c

```diff
@@ -157,6 +157,16 @@ OcMain (
     );
 }
 
+/*
+ * Leave a signature that wouldn't be stripped in release version
+ */
+static const char* opencore_revision __attribute__((used)) = "OpenCore revision: " OPEN_CORE_VERSION;
+/*
+ * Leave a marker for BootloaderChooser, to prevent confusion (an efi file from a folder EFI\OCXXX that would use file from EFI\OC)
+ */
+static const char* path_independant __attribute__((used)) = "path_independant";
```

Comment:
Are you going to bin-walk the entire driver for this string? Sorry, this sounds terrible. We have a custom DOS header signature for BOOTx64, I guess we could have one for OC too? It could include the version and you could check it against a minimum instead. CC @vit9696

Reply:
I intentionally did not include the version in OpenCore.efi, but in theory we could, as unlike `BOOTx64.efi` we do not need the exact match.